app/src/main/java/com/example/una/SurveyActivity.java

```diff
@@ -113,6 +113,8 @@ public void onClick(View v) {
                     Map<String, Object> user = new HashMap<>();
                     user.put("first_name", firstName);
                     user.put("last_name", lastName);
+                    user.put("fav_charity_ein", R.string.red_cross_ein);
```

Comment:
Add TODO to replace this later and create new issue?

Reply:
Yeah. We can either do that, or pick one randomly based on their interests, or we can let them change it manually by setting a new charity as their favorite on that charity's detail page